### ISSUE TITLE
Issues with Autopkg 2 - Oracle - VirtualBoxURLProvider

### ISSUE THREAD

**Issue by TSPARR** [autopkg/jessepeterson-recipes#90]
```
Traceback (most recent call last):
  File "/Library/AutoPkg/autopkglib/__init__.py", line 820, in get_processor
    _tmp = imp.load_source(processor_name, processor_filename)
  File "/Library/AutoPkg/Python3/Python.framework/Versions/Current/lib/python3.7/imp.py", line 171, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 696, in _load
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/munki/Library/AutoPkg/RecipeRepos/com.github.autopkg.jessepeterson-recipes/Oracle/VirtualBoxURLProvider.py", line 5, in <module>
    import urllib2
ModuleNotFoundError: No module named 'urllib2'
  File "/Library/AutoPkg/autopkglib/__init__.py", line 820, in get_processor
    _tmp = imp.load_source(processor_name, processor_filename)
Failed.
```

**Comment by jessepeterson** [comment]
Could you check that this was fixed with #86? Thanks!

**Comment by TSPARR** [comment]
This appears to have been resolved by the referenced PR.
